osu.Game/Screens/Play/HUD/HitErrorDisplay.cs

```diff
@@ -0,0 +1,198 @@
+                        RelativeSizeAxes = Axes.Y,
+                        Direction = FillDirection.Vertical,
+                    },
+                    new Container
+                    {
+                        Anchor = reversed ? Anchor.CentreRight : Anchor.CentreLeft,
+                        Origin = reversed ? Anchor.CentreRight : Anchor.CentreLeft,
+                        AutoSizeAxes = Axes.X,
+                        RelativeSizeAxes = Axes.Y,
+                        Child = arrow = new SpriteIcon
+                        {
+                            Anchor = Anchor.Centre,
+                            Origin = Anchor.Centre,
+                            RelativePositionAxes = Axes.Y,
+                            Icon = reversed ? FontAwesome.Solid.ChevronRight : FontAwesome.Solid.ChevronLeft,
+                            Size = new Vector2(8),
+                        }
+                    },
+                }
+            });
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            config.BindWith(OsuSetting.ScoreMeter, type);
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            HitWindows.SetDifficulty(beatmap.Value.BeatmapInfo.BaseDifficulty.OverallDifficulty);
+
+            bar.AddRange(new Drawable[]
+            {
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = ColourInfo.GradientVertical(colours.Yellow.Opacity(0), colours.Yellow),
+                    Height = (float)((HitWindows.Meh - HitWindows.Good) / (HitWindows.Meh * 2))
+                },
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colours.Green,
+                    Height = (float)((HitWindows.Good - HitWindows.Great) / (HitWindows.Meh * 2))
+                },
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colours.BlueLight,
+                    Height = (float)(HitWindows.Great / HitWindows.Meh)
+                },
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colours.Green,
+                    Height = (float)((HitWindows.Good - HitWindows.Great) / (HitWindows.Meh * 2))
+                },
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = ColourInfo.GradientVertical(colours.Yellow, colours.Yellow.Opacity(0)),
+                    Height = (float)((HitWindows.Meh - HitWindows.Good) / (HitWindows.Meh * 2))
+                }
+            });
+
+            type.BindValueChanged(onTypeChanged, true);
+        }
+
+        private void onTypeChanged(ValueChangedEvent<ScoreMeterType> type)
+        {
+            switch (type.NewValue)
```

Comment:
can be simplified into:
```csharp
this.FadeTo(type.NewValue == ScoreMeterType.HitError ? 1 : 0, fade_duration, easing);
```

Reply:
It looks better, but in the future we want to add another type, so not sure

Reply:
`type.NewValue > ScoreMeterType.None`?